Dockerfile

```diff
@@ -8,6 +8,13 @@ ENV CGO_ENABLED 0
 
 WORKDIR /go/src/github.com/minio/
 
+# add our user and group first with fixed uid and gid, this ensures
+# multiple updates to the minio image dont have permission issues.
+RUN addgroup -g 190 -S minio && adduser -u 190 -S -G minio minio
+
+# grab su-exec for easy step-down from root
+RUN apk add --no-cache 'su-exec>=0.2'
```

Comment:
Should this be in a new layer? why not add it to the next line below?

Reply:
yes, will add it as single layer